### PR TITLE
Repro error on isl null input

### PR DESCRIPTION
**Pull request by prigoyal**
relates to https://github.com/facebookresearch/TensorComprehensions/issues/124

error:
```
[prigoyal:~/local/TensorComprehensions: (master)]$ ./build/test/test_autotuner --gtest_filter=ATenCompilationUnitTest.Graph2 --debug_tc_mapper=true
Note: Google Test filter = ATenCompilationUnitTest.Graph2
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from ATenCompilationUnitTest
[ RUN      ] ATenCompilationUnitTest.Graph2
unknown file: Failure
C++ exception with description "/home/prigoyal/local/TensorComprehensions/third-party-install/include/isl/interface/isl.h:3185: NULL input" thrown in the test body.
[  FAILED  ] ATenCompilationUnitTest.Graph2 (2758 ms)
[----------] 1 test from ATenCompilationUnitTest (2758 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (2758 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] ATenCompilationUnitTest.Graph2

 1 FAILED TEST
```

### CODE REVIEW

**Comment by prigoyal** [comment]
I disabled the test and added a TODO with the issue number. should we land this once CI is green?

File: test/test_autotuner.cc [hunk 1]
```diff
@@ -89,6 +89,25 @@ struct ATenCompilationUnitTest : public ::testing::Test {
   }
 };
 
+// TODO: see #124
+TEST_F(ATenCompilationUnitTest, DISABLED_Graph2) {
```
Comment (ftynse) [inline test/test_autotuner.cc]: I'd prefer this as a non-autotuner test.  The problem is not specific to the autotuner, and introducing a ~1min test time does not look like a good idea.

Comment (prigoyal) [reply test/test_autotuner.cc]: @ftynse where shall this go? in particular, do we have a test file where we put such test cases? if not, I can create one.

**Comment by ftynse** [comment]
subsumed by #127

**Pull request closed without merging**